### PR TITLE
[WIP] Add binding calc

### PR DESCRIPTION
**Pull request by danieleongari**
This contains work chains to compute binding energies with FF and DFT.
Therefore, it is related to PR to:

* aiida-raspa: https://github.com/yakutovicha/aiida-raspa/pull/46
* aiida-lsmo: https://github.com/lsmo-epfl/aiida-lsmo/pull/23

### CODE REVIEW

**Comment by yakutovicha** [comment]
@danieleongari don't forget to add a simple run example here

**Comment by yakutovicha** [comment]
We were discussing with @danieleongari that it might be a good idea to keep the work chain in the [aiida-lsmo](https://github.com/lsmo-epfl/aiida-lsmo) repo for a while. When the work chain will be in a good shape and well tested, it can be moved here. 

The parser can still be modified of course.

We will have a meeting tomorrow to discuss this. @ltalirz let us know if you have any thoughts.

**Comment by ltalirz** [comment]
I think this is a good idea

File: aiida_cp2k/workchains/binding_energy.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+        # Store the workchain inputs in context (to be modified later)
+        self.ctx.base_inp = AttributeDict(self.exposed_inputs(Cp2kBaseWorkChain, 'cp2k_base'))
+
+        # Read yaml file selected as SinglefileData or chosen with the tag, and overwrite with custom modifications
+        if 'protocol_yaml' in self.inputs:
+            self.ctx.protocol = yaml.safe_load(self.inputs.protocol_yaml.open())
+        else:
+            thisdir = os.path.dirname(os.path.abspath(__file__))
+            yamlfullpath = os.path.join(thisdir, 'multistage_protocols', self.inputs.protocol_tag.value + '.yaml')
```
Comment (danieleongari) [inline aiida_cp2k/workchains/binding_energy.py]: @yakutovicha 
do you now a more elegant way to this? Such as open(aiida_cp2k.multistage_protocols.{TAG}.yaml) that works also if this work chain is in aiida-lsmo?

**Comment by danieleongari** [comment]
see https://github.com/aiidateam/aiida-cp2k/pull/70

**Pull request closed without merging**